vsts-extension-google-play.json

```diff
@@ -45,7 +45,7 @@
             "path": "_build/Tasks/GooglePlayPromote"
         },
         {
-            "path": "_build/Tasks/GooglePlayIncreaseRollout"
+            "path": "_build/Tasks/GooglePlayRolloutUpdate"
```

Comment:
Could you please describe more in description why do we need this change?

Reply:
Previously build path was determined using task name in task.json, but now we use the same name as the folder containing this task
Basically we need to do this because of [this change](https://github.com/microsoft/google-play-vsts-extension/pull/221/files#diff-53c70f37847db86d5ef8f8ed103140aebe630332facb6af5cd99e4845c180ef8R133), which was initially made to stay consistent with [azure-pipelines-tasks' make.js](https://github.com/microsoft/azure-pipelines-tasks/blob/master/make.js#L172)